lib/app/index.js

```diff
@@ -153,8 +153,8 @@ async function createApp (ssrContext) {
   <% if (store) { %>
   if (process.client) {
     // Replace store state before plugins execution
-    if (window.__NUXT__ && window.__NUXT__.state) {
-      store.replaceState(window.__NUXT__.state)
+    if (window.<%= globals.state %> && window.<%= globals.state %>.state) {
```

Comment:
Instead of `state.state` can't we use something like `context`?

Reply:
I like this suggestion, will change ASAP.